### DISCUSSION TITLE
Автобилд

### DISCUSSION THREAD

**Discussion by Aleks-Z** [theAsmodai/metamod-r#79] [Q&A] [answered]
Можешь сделать автобилд как у ReHLDS ? 
[Пример](http://nexus.rehlds.org/nexus/content/repositories/rehlds-dev/rehlds/rehlds/)
PS Возможно пригодиться, но это просто ребилд на CMake
[Metamod-P-CMake](https://github.com/SamVanheer/Metamod-P-CMake)

**Comment by hajimura** [comment]
Metamod-plugin isn't ready for now, i think it's too early to talk about "autobuild" as you say

**Comment by theAsmodai** [comment, marked as answer]
We already added this project to teamcity but it hidden for now.

**Comment by stamepicmorg** [comment]
maby close this issue? :)
